### DISCUSSION TITLE
Cannot use platform specific context in edge functions on a local development environment

### DISCUSSION THREAD

**Discussion by HenkBourgonje** [vercel/community#3552] [Help] [answered]
### Summary

I have a SvelteKit app deployed on Vercel in which I am using an Edge Function. In the Edge Function I am using the `event.platform.waitUntil()` functionality to return back to the application while the Edge Function runs in the background. This all works fine on the Vercel servers but when I try to run it locally using `vercel dev`, I cannot seem to use any platform specific context because `event.platform` keept being `undefined`. Any ideas how I could get this to work, or is this not possible at all?

### Example

https://github.com/HenkBourgonje/svelte-vercel-edge-local

### Steps to Reproduce

A simplified version of my Edge Function:
```ts
import type { Config } from '@sveltejs/kit';

export const config: Config = {
  runtime: 'edge'
};

export async function POST(event) {
  // platform is undefined here on local dev, though it does have the desired functionality on the Vercel servers.
  console.log("platform: ", event.platform)
  return Response.json({ message: 'ok' })
}
```

This is my `svelte.config.js`, as you can see I am using the correct adapter:
```js
import { vitePreprocess } from '@sveltejs/kit/vite';
import adapter from '@sveltejs/adapter-vercel'

/** @type {import('@sveltejs/kit').Config} */
const config = {
	preprocess: vitePreprocess(),

	kit: {
		adapter: adapter()
	}
};

export default config;
```

**Comment by amyegan** [comment]
Hey @HenkBourgonje. What version of Vercel CLI are you using?

**Reply by HenkBourgonje** [reply]
Hi @amyegan, I'm using version 31.2.2

**Reply by amyegan** [reply]
Would you mind trying with `31.2.3`?

**Reply by HenkBourgonje** [reply]
@amyegan I have tried it with `31.2.3` and it displays the same behaviour unfortunately.

**Reply by amyegan** [reply]
Thanks for trying that. If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I can share it with the CLI team for investigation.

**Reply by HenkBourgonje** [reply]
Okay, I've created it under the following branch. https://github.com/HenkBourgonje/svelte-vercel-edge-local
It just displays a button triggering the edge function and logging what `event.platform` is.

**Comment by Ethan-Arrowood** [comment, marked as answer]
Hi folks, we've received an answer to this internally. `event.platform` is unavailable in SvelteKit adapters currently, and since `context` does not exist either, `waitUntil` is unavailable during development in SvelteKit adapters currently.
